src/components/CautionaryContact/CautionaryContactView.js

```diff
@@ -1,6 +1,6 @@
 import BackButton from '../Layout/BackButton'
 import CautionaryContactTable from './CautionaryContactTable'
-import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlertsDescription'
+import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlerts'
 
 const CautionaryContactView = (searchQuery) => {
   const cautionaryContacts = CAUTIONARY_CONTACT
```

Comment:
We could just pass the `CAUTIONARY_CONTACTS` straight in to the component rather than define a new const for a slight readability improvement.

Reply:
I would recommend to use the name of `CAUTIONARY_ALERTS` as the `CAUTIONARY_CONTACTS` indicated only the `personal_alerts` but we need to include `location_alerts` as well

Reply:
Thank you both ;)